JulioC-19/LargeProjectMobile
# Julio/favorite api

Implemented authentication persistence, favorites api, added methods/data to the context and favorites screen.  
Context will allow every screen in the app to access the defined data and functions. 

To test: 
Login with:
crason.products@gmail.com
ucf
Navigate to the homescreen and tab the heart icon for any recipe, the background color should change. 
Navigate to the favorite screen, the prepopulated recipes should have the heart background red and the one selected before should be there as well. 
Tap a favorite recipe and refresh the page, the list should update and the recipe should be gone.
